config/environments/test.rb

```diff
@@ -11,6 +11,7 @@
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
   config.eager_load = false
+  config.action_mailer.default_url_options = { :host => "http://localhost:3000" }
```

Comment:
Looks like your fork may be a few commits behind master. I merged in a similar change to this file a few days ago. :)